bmv2/tests/openflow/Makefile.am

```diff
@@ -0,0 +1,38 @@
+if BMV2_TEST_ENV
+
+check_PROGRAMS = oftest
+
+oftest_SOURCES = oftest.c
+
+oftest_CPPFLAGS = -I$(includedir)
+oftest_CPPFLAGS += -I$(srcdir)/../thrift_endpoint
+oftest_CPPFLAGS += -I$(builddir)/../../oftest
+oftest_CPPFLAGS += -I$(top_srcdir)/pdfixed
+oftest_CPPFLAGS += -D_BMV2_
+
+if P4THRIFT
+    oftest_CPPFLAGS += -DP4THRIFT
+endif
+
+oftest_LDFLAGS = -L$(libdir)
+oftest_LDFLAGS += -pthread
+
+noinst_LTLIBRARIES = libthrift_endpoint.la
+libthrift_endpoint_la_SOURCES = ../thrift_endpoint/src/server.cpp
+
+oftest_LDADD = libthrift_endpoint.la
+oftest_LDADD += $(builddir)/../../libof.la -lp4ofagent $(builddir)/../../libpd.la -lbmpdfixed
+oftest_LDADD += -lruntimestubs -lsimpleswitch_thrift -lm -lstdc++ -lrt
+oftest_LDADD += $(THRIFT_LIB)
+
+TESTS = ./run_tests.sh
+
+else
+
+check-local:
```

Comment:
can't the `BMV2_TEST_ENV` check be placed in the parent directory's `Makefile.am` to avoid the code duplication, i.e. if it is not defined, just print this and do not use any `SUBDIRS`?